### PR TITLE
Adding ww package

### PR DESCRIPTION
**Pull request by Macelai**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
  - [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by igortg** [comment]
@marcelotrevisani - can you take a look? Some people at ESSS has the opinion that this is so simple that doesn't worth a conda-forge package. What are your thoughts?

**Comment by marcelotrevisani** [comment]
It is fine to create a package like that to be easier to add your scripts for the users. I am not opposed to that.
It might be simple indeed, but we also have some similar packages to this one. At the end of the day, it is more up to the maintainers this decision, if they want to add a package for it to be easier to install and use those scripts.. :)


I would personally go for a python implementation rather than a bash script and bat file. Or maybe something like ``xonsh``, but that is my opinion related to the project itself and not to the packaging of this.

**Comment by marcelotrevisani** [comment]
Would you like to proceed with this PR and get it merged?

@igortg , @Macelai

File: recipes/ww/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  home: https://github.com/ESSS/ww
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'The multiple-workspace batch script'
+
+extra:
+  recipe-maintainers:
+    - igortg
+    - loliveira
```
Comment (marcelotrevisani) [inline recipes/ww/meta.yaml:29]: @loliveira do you agree to maintain this recipe?

**Comment by igortg** [comment]
> 
> 
> Would you like to proceed with this PR and get it merged?
> 
> @igortg , @Macelai

Yes please. Thnks

File: recipes/ww/meta.yaml (hunk 1 above)
Comment (lvoliveira) [reply recipes/ww/meta.yaml:29]: yes

**Comment by Macelai** [comment]
> Would you like to proceed with this PR and get it merged?
> 
> @igortg , @Macelai

Yes, thanks

**Comment by igortg** [comment]
> I would personally go for a python implementation rather than a bash script and bat file. Or maybe something like `xonsh`, but that is my opinion related to the project itself and not to the packaging of this.

I don't know who is the original author (@tarcisiofischer?). But I always find it easier to manipulate and persist environment changes through bash/bat instead of Python.

File: recipes/ww/meta.yaml (hunk 1 above)
Comment (loliveira) [reply recipes/ww/meta.yaml:29]: No! Pls change loliveira to  lvoliveira =)

Comment (Macelai) [reply recipes/ww/meta.yaml]: Ops

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**